cluster/manifests/kube-state-metrics/rbac.yaml

```diff
@@ -3,16 +3,130 @@ kind: ServiceAccount
 metadata:
   name: kube-state-metrics
   namespace: kube-system
+  labels:
+    application: kube-state-metrics
 ---
+apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: kube-state-metrics
+  labels:
+    application: kube-state-metrics
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: system:monitor
+  name: kube-state-metrics
```

Comment:
We need to fix this before we can merge, otherwise we get an error when CLM tries to apply it to existing clusters.

Reply:
Added a deletion to fix it. It's not perfect but good enough.